libvmaf/include/libvmaf/model.h

```diff
@@ -32,12 +32,29 @@ enum VmafModelFlags {
 
 typedef struct VmafModelConfig {
     char *name;
-    char *path;
     uint64_t flags;
 } VmafModelConfig;
 
-int vmaf_model_load_from_path(VmafModel **model, VmafModelConfig *cfg);
+int vmaf_model_load_from_path(VmafModel **model, VmafModelConfig *cfg,
+                              const char *path);
 
 void vmaf_model_destroy(VmafModel *model);
 
+typedef struct VmafModelCollection VmafModelCollection;
+
+typedef struct VmafModelCollectionScore {
```

Comment:
Can we rename to be something like `VmafBootstrapModelCollectionScore`? Since bagging, p95, lo, hi are all bootstraping related.